scipy/sparse/linalg/eigen/arpack/arpack.py

```diff
@@ -1446,7 +1447,13 @@ def eigsh(A, k=6, M=None, sigma=None, which='LM', v0=None,
     OPinv : N x N matrix, array, sparse matrix, or LinearOperator
         See notes in sigma, above.
     return_eigenvectors : bool
-        Return eigenvectors (True) in addition to eigenvalues
+        Return eigenvectors (True) in addition to eigenvalues. This value determines
+        the order in which eigenvalues are sorted.
+
+            If `return_eigenvectors` is True, eigenvalues are sorted by algebraic value
+
+            If `return_eigenvectors` is False, eigenvalues are sorted by magnitude value
```

Comment:
The term "absolute value" sounds more natural.

Reply:
I agree with you.
Generally speaking, 'absolution' is better that 'magnitude'.
Thanks for your quickly reviews, @ilayn.